docs/reference.md

```diff
@@ -15,8 +15,7 @@ sylius_resource:
                     validation_groups: [sylius]
             options:
                 object_manager: default
-            templates:
-                form: Book/_form.html.twig
+            templates: 'book' # Templates namespace for this resource, default value is empty
```

Comment:
We should perhaps be more specific about what's expected here ? Like `Book/Crud/` 
WDYT ?

Reply:
I mean, just this `book` may be too simple for the users to understand what's expected

Reply:
👍 for `Book/Crud/`. Additionally, I would maybe change the comment to `Templates **directory** (...)`? As templates do not have namespace _per se_, as e.g. php classes 😄 We can even go wth some example, like:

```yaml
templates: 'Book/Crud/' # directory of templates, this would be placed in `templates/Book/Crud/`. Default value is empty
```

or something similar, wdyt?

Reply:
@Zales0123 I'm ok, but using snake case, this is the Symfony best practice recommandation.
`templates: 'book/crud/' # directory of templates, this would be placed in `templates/book/crud/`. Default value is empty`

Reply:
I agree with that, but it is not what Sylius does everywhere for now.
Perhaps it is time to initiate that change :smile:

Reply:
For Sylius, I don't know, but this package is not only used on Sylius projects, so, I think we should use the recommandation.

Reply:
perhaps we can add two recommendations? 
```suggestion
            templates: 'book' # directory of templates, this would be placed in `templates/book/`. Default value is empty. Keep in mind, that in Sylius we usually use UpperCamelCase notation in contrary to Symfony recommendations
```